src/packages/@workspaces/features/save-as-request/layout/SaveAsRequest.svelte

```diff
@@ -841,7 +841,8 @@
         }}
         title={"+ Collection"}
         buttonClassProp={"btn mb-2"}
-        buttonStyleProp={"color: var(--text-primary-300); font-size: var(--base-text); border: 1px solid var(--border-primary-200);"}
+        type={"teritiary"}
+        buttonStyleProp={"font-size: var(--base-text);"}
       />
     {:else if path.length > 0 && path[path.length - 1].type === ItemType.COLLECTION}
       <Button
```

Comment:
could you please make a new button component in save-as-request/components/button/Button.svelte
and revert your type `tertiary`  changes from @library/ui/button/Button.svelte